libraries/core-react/src/components/Input/Input.tsx

```diff
@@ -9,10 +9,9 @@ import {
 } from '../../utils'
 import type { Variants } from '../TextField/types'
 import { useEds } from '../EdsProvider'
+import { useToken } from '../../hooks'
 
-const { input, inputVariants } = tokens
-
-const Variation = ({ variant, token, density }: StyledProps) => {
+const StyledInput = styled.input(({ variant, theme }: StyledProps) => {
   if (!variant) {
     return ``
   }
```

Comment:
Suggest we remove this as we merged all css into one class and this escape is redundant now.